_data/hosting.yml

```diff
@@ -345,6 +345,13 @@ websites:
       - totp
     doc: https://www.netcup-wiki.de/wiki/Stammdaten_CCP#Zwei-Faktor-Authentifizierung_.282FA.29
 
+  - name: Netlify
+    url: https://www.netlify.com
+    twitter: Netlify
+    facebook: netlify
+    img: netlify.png
+    status: https://community.netlify.com/t/feature-request-two-factor-authentication-2fa-on-accounts/2876
```

Comment:
The status tag is only meant to be used when a site is actively working on implementing 2FA. From reading the thread it looks like they're not working on it at the moment.
```suggestion
```